sympy/core/tests/test_expr.py

```diff
@@ -1846,8 +1846,8 @@ def test_issue_7426():
 
 
 def test_issue_1112():
-    x = Symbol('x', extended_positive=False)
-    assert (x > 0) is S.false
+    # x = Symbol('x', extended_positive=False)
+    # assert (x > 0) is S.false
```

Comment:
Can you correct the issue number here and add a comment referring back to this PR that explains why that test shouldn't pass?

Reply:
Can you correct the issue number here?

Also we need a test for this so let's have one with `x = Symbol('x', extended_real=True, extended_positive=False)` and another with `extended_positive=False` that tests the new behaviour like
```
assert unchanged(Gt, x, 0)
```
```